package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "ffc-pay-batch-processor",
-  "version": "1.24.0",
+  "version": "1.24.1",
   "description": "FFC Payment Batch Processor",
   "homepage": "https://github.com/DEFRA/ffc-pay-batch-processor",
   "main": "app/index.js",
   "scripts": {
     "pretest": "npm run test:lint",
-    "test": "jest --runInBand",
+    "test": "jest --runInBand && npm run test:acceptance",
+    "test:acceptance": "cd test/acceptance && cucumber-js",
```

Comment:
Suggest changing this to use the `--config` argument rather than navigating through the filesystem.  That way there is only one command to manage.